userguide/drawing_classifier/export-coreml.md

```diff
@@ -167,6 +162,39 @@ func rasterize(drawing stroke_based_drawing:Drawing) -> CGImage {
         space:grayscale, bitmapInfo:CGImageAlphaInfo.none.rawValue)
     let final_rect = CGRect(x: 0.0, y: 0.0, width: 28.0, height: 28.0)
     final_bitmap_context?.draw(intermediate_image!, in: final_rect)
-    return (final_bitmap_context?.makeImage())!
+    return cgImage : (final_bitmap_context?.makeImage())!
```

Comment:
Why do we need to add the `cgImage` annotation here?

Reply:
Left it by mistake